Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -1058,109 +1058,204 @@ theorem comp_familyOfBFamily {o} (f : ∀ a < o, α) (g : α → β) :
 
 /-! ### Supremum of a family of ordinals -/
 
--- Porting note: Universes should be specified in `sup`s.
-
 /-- The supremum of a family of ordinals -/
+
+@[deprecated iSup (since := "2024-08-27")]
 def sup {ι : Type u} (f : ι → Ordinal.{max u v}) : Ordinal.{max u v} :=
   iSup f
 
-@[simp]
+set_option linter.deprecated false in
+@[deprecated (since := "2024-08-27")]
 theorem sSup_eq_sup {ι : Type u} (f : ι → Ordinal.{max u v}) : sSup (Set.range f) = sup.{_, v} f :=
   rfl
 
 /-- The range of an indexed ordinal function, whose outputs live in a higher universe than the
-    inputs, is always bounded above. See `Ordinal.lsub` for an explicit bound. -/
-theorem bddAbove_range {ι : Type u} (f : ι → Ordinal.{max u v}) : BddAbove (Set.range f) :=
-  ⟨(iSup (succ ∘ card ∘ f)).ord, by
-    rintro a ⟨i, rfl⟩
-    exact le_of_lt (Cardinal.lt_ord.2 ((lt_succ _).trans_le
-      (le_ciSup (Cardinal.bddAbove_range.{_, v} _) _)))⟩
-
+inputs, is always bounded above. See `Ordinal.lsub` for an explicit bound. -/
+theorem bddAbove_range {ι : Type u} (f : ι → Ordinal.{max u v}) : BddAbove (Set.range f) := by
+  use (iSup (succ ∘ card ∘ f)).ord
+  rintro a ⟨i, rfl⟩
+  exact (Cardinal.lt_ord.2 ((lt_succ _).trans_le
+    (le_ciSup (Cardinal.bddAbove_range.{u, v} _) _))).le
```

Comment:
This didn't need changing, right? Maybe keep the change for the next refactor, where you will be replacing `{ι : Type u}` by `{ι : Type*} [Small.{u} ι]`

Reply:
Yep, that's the idea. I'm getting this done for `Cardinal` in the meanwhile.